src/util/OptionalBlueprint.tsx

```diff
@@ -1,36 +1,11 @@
-import type { Classes } from '@blueprintjs/core';
+import { Classes } from '@blueprintjs/core';
 import type { IconNames } from '@blueprintjs/icons';
 import classNames from 'classnames';
 import _ from 'lodash';
 import * as React from 'react';
 
-// Performs a top level import of blueprint and gets the classes.
-async function getBluePrintClasses() {
```

Comment:
Keeping it optional would be nice - why did you end up removing this? Lack of support in build tools?

Reply:
Top level async statements are not supported when the target is `es2017`; the library would have to be bumped up to `es2020` in order to take advantage of that.

Without top level awaits (and support for `import` to be called as a function), this approach wouldn't work.

If you're willing to have the module/lib updated to `es2020`, I'd be happy to make that change and include this back into the PR.